src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -35,34 +13,19 @@ interface Props {
 }
 
 const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc", currentUser }: Props) => {
-  const classes = useStyles()
   const { basePath, query } = useRouter()
-
-  let searchParams = new URLSearchParams(encode(query))
-  searchParams = deleteQueryParamsByName(["unlockException", "unlockTrigger"], searchParams)
-
   const orderByParams = (orderBy: string) => `${basePath}/?${new URLSearchParams({ ...query, orderBy, order })}`
-  const caseDetailsPath = (id: number) => `${basePath}/court-cases/${id}`
-  const unlockCaseWithReasonPath = (reason: "Trigger" | "Exception", caseId: string) => {
-    searchParams.append(`unlock${reason}`, caseId)
-    return `${basePath}/?${searchParams}`
-  }
-
-  const canUnlockCase = (lockedUsername: string): boolean => {
-    return currentUser.groups.includes("Supervisor") || currentUser.username === lockedUsername
-  }
-
   const tableHead = (
     <Table.Row>
       <Table.Cell></Table.Cell>
-      <Table.CellHeader>
+      <Table.CellHeader setWidth={"178px"}>
```

Comment:
Why is it `178px` specifically? Would it be worth sticking it in a variable with a meaningful name for visibility?

Reply:
It is 178px in the design and without it the header wraps into 2 lines due to `flex`ing. I think it will only be used here once so putting it in a global variable may not be required.